lib/pbxProject.js

```diff
@@ -825,8 +825,8 @@ pbxProject.prototype.addTargetDependency = function (target, dependencyTargets)
     var pbxTargetDependencySection = this.hash.project.objects[pbxTargetDependency];
     var pbxContainerItemProxySection = this.hash.project.objects[pbxContainerItemProxy];
 
-    for (var index = 0; index < dependencyTargets.length; index++) {
-        var dependencyTargetUuid = dependencyTargets[index];
+    for (var index2 = 0; index2 < dependencyTargets.length; index2++) {
```

Comment:
Violations like these would disappear later when transforming var to let. Having `index2` as a variable name could be irritating.